Database/Corrections/TBC/tbcQuestFixes.lua

```diff
@@ -1327,6 +1340,9 @@ function QuestieTBCQuestFixes:Load()
                 [zoneIDs.IRONFORGE]={{70.5,89.56},},
             }},
         },
+        [11383] = {
+            [questKeys.objectives] = {nil,nil,nil,nil,{{17839,},17839},},
```

Comment:
```suggestion
            [questKeys.objectives] = {{17839,},nil,nil,nil,nil,},
```

No need to add a single NPC objective as killCredit objective 👍🏻